code/__DEFINES/loadout.dm

```diff
@@ -526,5 +526,7 @@ GLOBAL_LIST_INIT(loadout_job_supported, list(
 ///The maximum number of loadouts one player can have
 #define MAXIMUM_LOADOUT 50
 
-/// The currently accepted loadout version, all other loadouts will be erased from savefiles
-#define CURRENT_LOADOUT_VERSION 5
+/// The current loadout version
+#define CURRENT_LOADOUT_VERSION 6
+
+GLOBAL_LIST_INIT(accepted_loadout_versions, list(5,6))
```

Comment:
Why not just do it how savefiles already does?

Reply:
We accept version 5, because we have a method of transforming from 5 to 6. 
This was not the case prior